man/swaync.5.scd

```diff
@@ -415,6 +435,16 @@ config file to be able to detect config errors
 								type: string ++
 								default: "" ++
 								description: "Command to be executed on click" ++
```

Comment:
We should have a way of getting the toggle state through an environment variable

Reply:
Added the code for this. Still have to update the documentation for this..

Also might add a option to set the toggle active state depending on the output of a script on_cc_visibility_change.

And I noticed that there might be breaking changes for users who use commands like "\~/script.sh" (me :upside_down_face: ) since "\~" is not recognized anymore with the Process.spawn_async method :)